edxlearndot/models.py

```diff
@@ -14,7 +14,7 @@ class CourseMapping(models.Model):
     learndot_component_id = models.BigIntegerField(help_text="The numeric ID of the Learndot component.")
     edx_course_key = CourseKeyField(max_length=255, db_index=True, help_text="The edX course ID.")
 
-    class Meta(object):
+    class Meta(object):  # pylint: disable=useless-object-inheritance
```

Comment:
nit: python3 classes all inherit from object:
```suggestion
    class Meta(object):
```